Il2CppInterop.Generator/Runners/InteropAssemblyGenerator.cs

```diff
@@ -67,29 +67,39 @@ public void Run(GeneratorOptions options)
             Pass10CreateTypedefs.DoPass(rewriteContext);
         }
 
+        using (new TimingCookie("Computing generic parameter usage"))
+        {
+            Pass11ComputeGenericParameterSpecifics.DoPass(rewriteContext);
+        }
+
         using (new TimingCookie("Computing struct blittability"))
         {
-            Pass11ComputeTypeSpecifics.DoPass(rewriteContext);
+            Pass12ComputeTypeSpecifics.DoPass(rewriteContext);
+        }
+
+        using (new TimingCookie("Creating unboxed struct types"))
+        {
+            Pass13CreateGenericNonBlittableTypes.DoPass(rewriteContext);
         }
 
         using (new TimingCookie("Filling typedefs"))
         {
-            Pass12FillTypedefs.DoPass(rewriteContext);
+            Pass14FillTypedefs.DoPass(rewriteContext);
         }
 
         using (new TimingCookie("Filling generic constraints"))
         {
-            Pass13FillGenericConstraints.DoPass(rewriteContext);
+            Pass15FillGenericConstraints.DoPass(rewriteContext);
         }
 
         using (new TimingCookie("Creating members"))
         {
-            Pass15GenerateMemberContexts.DoPass(rewriteContext);
+            Pass16GenerateMemberContexts.DoPass(rewriteContext);
         }
 
         using (new TimingCookie("Scanning method cross-references"))
         {
-            Pass16ScanMethodRefs.DoPass(rewriteContext, options);
+            Pass17ScanMethodRefs.DoPass(rewriteContext, options);
         }
```

Comment:
Please make this part of pass 11 instead of renaming the other passes. It fits the theme of `ComputeTypeSpecifics`.

Reply:
Sure. I just felt that `ComputeTypeSpecifics` step code size was growing too big